taxi/models.py

```diff
@@ -18,6 +18,7 @@ class Driver(AbstractUser):
     license_number = models.CharField(max_length=255, unique=True)
 
     class Meta:
+        ordering = ["username"]
```

Comment:
don't need to change ordering for a full model, you can change it for you view if you need it